kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/OperatorChangeDetectionST.java

```diff
@@ -221,7 +220,7 @@ void shouldUpdateDeploymentWhenKafkaProxyChanges(String namespace) {
         // @formatter:off
         KafkaProxyBuilder updatedKafkaProxy = kafkaProxy.edit()
                 .editOrNewSpec()
-                .withPodTemplate(new PodTemplateSpecBuilder().build())
+                .withReplicas(2)
```

Comment:
errr this shouldn't cause the kind of change the test was looking for, and I'd talked myself out of this triggering a change to the checksum. 

This might work for now (i.e. the scope of this merge) but scaling the deployment shouldn't trigger a roll of existing pods and that needs fixed.

Reply:
with the other MR merged I can make a change to the infrastructure instead